tests/sflow_test.py

```diff
@@ -0,0 +1,72 @@
+import os
+import sys
+import pytest
+from click.testing import CliRunner
+from utilities_common.db import Db
+
+test_path = os.path.dirname(os.path.abspath(__file__))
+modules_path = os.path.dirname(test_path)
+sys.path.insert(0, test_path)
+sys.path.insert(0, modules_path)
```

Comment:
are these necessary? can you remove if they are not necessary

Reply:
done.